lib/tasks/massage_types.rake

```diff
@@ -0,0 +1,12 @@
+namespace 'massage_types' do
+  desc 'Create Massage Types'
+  task :seed => :environment do
+    types = [['Deep Tissue', 60], ['Swedish', 90], ['Hot Stone', 120], ['Shiatsu', 120]]
```

Comment:
It is useful for such things to not store User friendly names in the database.  I recommend something like 'deep_tissue', 'swedish', 'hot_stone', 'shiatsu'.  Then you translate them using en.yml.